### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by Maihav**
**Update README file on Basic-Bot sample**

We updated the **basic-bot** README file to match with the current implementation.

As you can see below, the previous version of the README used a **.bot** file. As this sample does not include it, that made the README out-of-date.  
 
image

To improve the user’s experience when executing the current sample, we suggested a way to run it by updating the _appsetting_ file. Also, we included a step-by-step to obtain those keys on Azure and LUIS portal.

Now, the README uses the _appseting_ configuration to run the basic-bot. You can find a preview of the improvement below. 
 
image

### CODE REVIEW

File: samples/csharp_dotnetcore/13.basic-bot/README.md [hunk 1]
```diff
@@ -14,111 +15,66 @@ git clone https://github.com/Microsoft/botbuilder-samples.git
...
+        "LuisAPIHostName": "<YOUR LUIS HOST NAME>"
+    }
+    ```
 # Running Locally
 
 ## Visual Studio
-- Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/13.basic-bot`) and open `BasicBot.csproj` in Visual Studio.
-- Run the project (press `F5` key)
+1. Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/13.basic-bot`) and open `CoreBot.csproj` in Visual Studio
+3. Run the project (press `F5` key)
```
Comment (denscollo) [inline samples/csharp_dotnetcore/13.basic-bot/README.md]: I think this'd be a 2. instead of 3., right?

Comment (Maihav) [reply samples/csharp_dotnetcore/13.basic-bot/README.md]: Done

**Review by denscollo — APPROVED** [review]
LGTM 👍

**Review by JuanAr — APPROVED** [review]
Changes look good to me. Promote to MS and assign to Chris if possible.
Add a bit more context in the PR description.

**Comment by denscollo** [comment]
Close due to a comment of Chris saying that there's a team already working on this.

**Pull request closed without merging**